src/rpcgovernance.cpp

```diff
@@ -623,6 +623,18 @@ UniValue voteraw(const UniValue& params, bool fHelp)
 
 UniValue getgovernanceinfo(const UniValue& params, bool fHelp)
 {
+    int nLastSuperblock, nNextSuperblock;
+
+    // Get current block height
+    int nBlockHeight = (int)chainActive.Height();
+
+    // Get chain parameters
+    int nSuperblockStartBlock = Params().GetConsensus().nSuperblockStartBlock;
+    int nSuperblockCycle = Params().GetConsensus().nSuperblockCycle;
+
+    // Get first superblock
+    int nFirstSuperblock = nSuperblockStartBlock - (nSuperblockStartBlock % nSuperblockCycle) + nSuperblockCycle;
```

Comment:
Yep, I agree with @tgflynn here, that's why I had `nFirstSuperblockOffset` in my code sample there https://github.com/dashpay/dash/pull/982#issuecomment-244577705